Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/exchange/transform/uddi/UDDITransform.java

```diff
@@ -215,50 +215,50 @@ private static void buildTargetRegions(BusinessEntity entity, OrganizationType o
         }
     }
 
-    private static List<String> buildEmails(Contact contact) {
-        List<String> emails = null;
+    private static void buildEmails(Contact contact, ContactType cType) {
+        List<String> emails;
         if (contact != null && CollectionUtils.isNotEmpty(contact.getEmail())) {
```

Comment:
Optional:  contact will never be null in this case since for loop above guarantee

Reply:
contact is an element in the list. It could be null and that's why a null check.